x/staking/spec/01_state.md

```diff
@@ -92,6 +98,24 @@ type Validator struct {
     MinSelfDelegation       sdk.Int         // validator's self declared minimum self delegation
 }
 
+//// Epoching change fields
+// MinSelfDelegation, Commission
+
+//// Endblocker change fields
+// Jailed, 
+
+//// Instant change fields
+// DelegatorShares, 
```

Comment:
Why is this instance change?

Reply:
DelegatorShare won't be part of epoching but when user run withdraw, it will take effect instantly. (or on validator update)